.changeset/thirty-paws-rush.md

```diff
@@ -0,0 +1,16 @@
+---
+'@shopify/ui-extensions': minor
+---
+
+- Added customer account migration guide with comparison tables mapping legacy components to new Polaris web components
+- Removed extended form field event properties to simplify component APIs
```

Comment:
This was just in the docs, right? I would clarify that because a developer might see _Removed_ and think something was removed from the package.